REMAINING.PUBLIC.md

```diff
@@ -0,0 +1,70 @@
+### Remaining public aliases
+
+annotation_reader
+cache.app_clearer
+$commandId
+debug.templating.engine.php
+profiler.request_matcher
+router
+security.access.decision_manager
+security.acl.cache
+security.acl.dbal.connection
+security.acl.provider
+security.encoder_factory
+security.firewall
+security.password_encoder
+session.handler
+session.storage
+session.storage.filesystem
+templating
+templating.loader
+translator
+twig.loader
+
+### Remaining public services
+
+cache.adapter.* <- pools are private by default when created by config so not a pb
+cache.app
+cache_clearer <- for the cache:clear command
+cache.global_clearer <- for the cache:pool:clear command to clear all pools
+cache.system
+cache_warmer <- required to bootstrap the kernel
+data_collector.cache
+data_collector.dump
+event_dispatcher <- required to wire console apps
+file_locator
+filesystem
+form.factory
+form.type.* <- deprecated
+http_kernel <- required to bootstrap the kernel
+kernel
+profiler <- used in tests
+property_accessor
+property_info
+request_stack
+routing.loader <- used by routing
+security.acl.dbal.schema
+security.authentication_utils
+security.authorization_checker
+security.csrf.token_manager
+security.firewall
+security.token_storage
+security.validator.user_password
+serializer
+session
+state_machine.abstract <- state machines created by config are public
+workflow.abstract <- workflows created by config are public
```

Comment:
can't we make them private instead ? The documented way to get them is to use the registry AFAIK, not to get the service from the container